### PR TITLE
Add test for open base dir and remove supression

### PR DESCRIPTION
**Pull request by exussum12**

### CODE REVIEW

**Comment by exussum12** [comment]
Adds a test in for open base dir and removes error suppression to handle the restrictions

**Comment by GrahamCampbell** [comment]
Hmmm. What's wrong with keeping the error suppression? This seems more hacky than the error suppression to me?

**Comment by exussum12** [comment]
Can do. The test included would pass with just error suppression.

**Comment by Seldaek** [comment]
Yeah I'd tend to say let's leave it as is and close this PR..

**Comment by exussum12** [comment]
Updated to just be the test, the test would still be useful

**Comment by Seldaek** [comment]
Sure thing, thanks

File: tests/CaBundleTest.php [hunk 1]
```diff
@@ -101,6 +101,16 @@ public function testIsOpensslParseSafeFalse()
...
     }
 
+    public function testOpenBaseDir()
+    {
+        $oldValue = ini_get('open_basedir');
+        ini_set('open_basedir', realpath(__DIR__ . '/../'));
+        $certFilePath = CaBundle::getSystemCaRootBundlePath();
+        $validResult = CaBundle::validateCaFile($certFilePath, null);
+        $this->assertTrue($validResult);
+        ini_set('open_basedir', $oldValue);
```
Comment (GrahamCampbell) [inline tests/CaBundleTest.php]: This should be done in a try-finally block, otherwise if the assertion fails, this code is never run?

Comment (exussum12) [reply tests/CaBundleTest.php]: Could flip the two around. Set it back before the assertion ?

Comment (GrahamCampbell) [reply tests/CaBundleTest.php]: Ok, what if the other code crashes?

Comment (exussum12) [reply tests/CaBundleTest.php]: Ini set only effects the current thread. If it crashes imagine the test suite would crash and wouldn't need setting back ?

Comment (GrahamCampbell) [reply tests/CaBundleTest.php]: The problem is it will mess up the other tests.

Comment (Seldaek) [reply tests/CaBundleTest.php]: I appreciate the enthusiasm  here but it's a tiny lib with a tiny scope, and this has a 1% chance of going bad, leading to a 1% of confusing tests.. let's please move on.

**Pull request merged**